include/solvers/cahn_hilliard.h

```diff
@@ -71,12 +71,11 @@ class CahnHilliard : public AuxiliaryPhysics<dim, TrilinosWrappers::MPI::Vector>
     if (simulation_parameters.mesh.simplex)
       {
         // for simplex meshes
-        const FE_SimplexP<dim> phase_order_fe(
-          simulation_parameters.fem_parameters.phase_ch_order);
-        const FE_SimplexP<dim> potential_fe(
-          simulation_parameters.fem_parameters.potential_ch_order);
-        fe =
-          std::make_shared<FESystem<dim>>(phase_order_fe, 1, potential_fe, 1);
+        fe = std::make_shared<FESystem<dim>>(
+          FE_Q<dim>(simulation_parameters.fem_parameters.phase_ch_order),
+          1,
+          FE_Q<dim>(simulation_parameters.fem_parameters.potential_ch_order),
+          1);
```

Comment:
I think this is a mistake, no? You are supposed to have `FE_SimplexP `instead of only `FE_Q.`.. or you are not going to support simplex meshes?

Reply:
It was a definitely a bad copy/paste, thanks